pinot-segment-local/src/main/java/org/apache/pinot/segment/local/indexsegment/mutable/MutableSegmentImpl.java

```diff
@@ -1211,17 +1230,29 @@ private static Map<String, Pair<String, ValueAggregator>> fromAggregationConfig(
           "aggregation function must be a function: %s", config);
       FunctionContext functionContext = expressionContext.getFunction();
       TableConfigUtils.validateIngestionAggregation(functionContext.getFunctionName());
-      Preconditions.checkState(functionContext.getArguments().size() == 1,
-          "aggregation function can only have one argument: %s", config);
+
```

Comment:
IMO checking whether there are `>= 1` arguments is enough. Seems the check is already applied in the `TableConfigUtils`, so we can actually remove this check

Reply:
you're right, this is done in table utils, i'll remove it from here